tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -512,15 +512,15 @@ stages:
         parameters:
           arch: 'aarch64'
           machine_pool: 'aiinfra-linux-ARM64-CPU-2019'
-          base_image: 'arm64v8/ubuntu:18.04'
+          base_image: 'arm64v8/ubuntu:20.04'
           device: '-d AZURE'
```

Comment:
It would not magically work. The build scripts we use do not support Ubuntu. We need to either patch the scripts to add Ubuntu support, or give up on it. Last year I asked the team whether they prefer Ubuntu more or better C++ standard support. They choose C++. So, going forward we should use manylinux_2_28 which is based on centos8.  See here: https://github.com/pypa/manylinux/blob/bd00b6b4ef3f876172d09c02d065c6440c468f78/build.sh#L44 I think we can remove this part and consider upgrading our manylinux standard from 2_17 to 2_28. I think it won't harm ARM usages, because centos 7 is much older than ARM64 CPUs. I don't see anyone run an ARM64 application on CentOS7. We can discuss with the team if we want to upgrade all of them to 2_28 or just ARM64 packages in the upcoming release. Or we just delete the Ubuntu part and wait.  The deadline of the upgrade work is June 30, 2024(when CentOS7 becomes EOL). Sooner is better because we will get newer compilers so that we can ship our fp16 features better.